pkg/controller/quayecosystem/resources/storage.go

```diff
@@ -50,6 +50,7 @@ func GetDatabasePVCDefinition(meta metav1.ObjectMeta, volumeSize string) *corev1
 					corev1.ResourceName(corev1.ResourceStorage): resource.MustParse(volumeSize),
 				},
 			},
+			StorageClassName: storageClass,
 		},
```

Comment:
Take a look at the method above for how the StorageClass is being added to the object. It should only be added if the value is defined and is not an empty string